proposals/hasCrossSiteAncestor.md

```diff
@@ -115,6 +114,7 @@ A Promise that will be fulfilled with a `Cookie.partitionKey` object that matche
 ##### Error conditions
 - If host permissions are not granted for the document whose partitionkey is getting queried an error will be returned.
 - When the parameters passed do not correspond to an existing frame, an error will be returned.
+- If the partitionKey that would be associated with the frame can not be serialized an error will be returned. This can happen when the origin associated with the topLevelSite is opaque or if the underlying key associated with the frame has a nonce. 
```

Comment:
Is this equivalent to getting an error when `document.cookie` is read/set from the web page? If not, then I would still expect a way to manage the cookies for that document.

If it is, it may be worth calling out that there cookies cannot exist in such documents.

Reply:
Yes, it would be equivalent to getting an error when `document.cookie` is read/set from a web page.

My thought is that having an unserializable `partitionKey` return an error will align us with the behavior in the [HTML spec](https://html.spec.whatwg.org/multipage/dom.html#cookie-averse-document-object) where it describes throwing an error is the origin of a document is opaque.

Reply:
When I visit `data:text/html,<script>alert(document.cookie)</script>`, it throws in Chrome but does not in Firefox. I'd therefore like to emphasize that the behavior matches the usual behavior on the web platform, so that the `cookies` extension API can always manage all cookies that can be created by websites, and not a subset of them.

Reply:
Resolving based on this [commit](https://github.com/w3c/webextensions/pull/712/commits/eb67586a416514a041e0f2658f88e24746489f80).